dm/config/task_converters.go

```diff
@@ -308,6 +310,8 @@ func OpenAPITaskToSubTaskConfigs(task *openapi.Task, toDBCfg *dbconfig.DBConfig,
 		if task.IgnoreCheckingItems != nil && len(*task.IgnoreCheckingItems) != 0 {
 			subTaskCfg.IgnoreCheckingItems = *task.IgnoreCheckingItems
 		}
+		// set syncer IO total bytes counter
+		subTaskCfg.InitIOCounters()
```

Comment:
should we update SubTaskConfigsToTaskConfig func?

Reply:
I think SubTaskConfigsToTaskConfig is used to get config by gRPC and OpenAPI. And i think the two IO counter is read only and not a real config🤔. So it is not necessary convert to TaskConfig